src/test/java/com/jcabi/email/EnvelopeTest.java

```diff
@@ -70,6 +73,43 @@ public void cachesPreviousCall() throws Exception {
     }
 
     /**
+     * Verifies that the encoding of the mail is set correctly and - as a
```

Comment:
Good description, but I think it is too long, let's go with:  `Envelope should handle non-Latin characters.`